From 99624dbc088d24b85d8326cd5cc1d69ebc648f85 Mon Sep 17 00:00:00 2001
From: Kevin Hao <kexin.hao@windriver.com>
Date: Sun, 28 Sep 2014 10:38:42 +0800
Subject: [PATCH] cpuidle: disable the nap mode for ls1021a

The current cpuidle driver for ls1021a SoC can't work in nap mode,
so disable it temporarily.

Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/cpuidle/cpuidle-ls1.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/cpuidle/cpuidle-ls1.c b/drivers/cpuidle/cpuidle-ls1.c
index 68234e4..e659c58 100644
--- a/drivers/cpuidle/cpuidle-ls1.c
+++ b/drivers/cpuidle/cpuidle-ls1.c
@@ -74,6 +74,7 @@ static struct cpuidle_driver ls1_cpuidle_driver = {
 			.enter = ls1_enter_cluster_stop,
 			.name = "nap",
 			.desc = "Cluster Stop Mode",
+			.disabled = true,
 		},
 	},
 	.state_count = LS1_CPUIDLE_MAX_STATES,
-- 
2.0.2

